x-pack/metricbeat/module/autoops_es/metricset/metricset.go

```diff
@@ -9,11 +9,23 @@ import (
 
 	"github.com/elastic/beats/v7/metricbeat/mb"
 	"github.com/elastic/beats/v7/metricbeat/module/elasticsearch"
+	"github.com/elastic/beats/v7/x-pack/metricbeat/module/autoops_es/ccm"
 	"github.com/elastic/beats/v7/x-pack/metricbeat/module/autoops_es/events"
 	"github.com/elastic/beats/v7/x-pack/metricbeat/module/autoops_es/utils"
+	"os"
+	"sync"
```

Comment:
Move these to the top (or go into the `x-pack/metricbeat` and run `make update`).